src/js/overlay.js

```diff
@@ -201,8 +201,6 @@ Overlay.prototype.show = function() {
 		document.body.appendChild(shadow);
 	}
 
-	this.content.focus();
```

Comment:
That's what I thought would happen but didn't get that behaviour. Will look at accessibility after fixing bugs